File: Backend/src/Fibertest30.Application/Users/Entities/UserSettings.cs
namespace Fibertest30.Application;

public class UserSettings
{
    public string UserId { get; set; } = null!;
    public string Theme { get; set; } = string.Empty;
    public string Language { get; set; } = string.Empty;
    public string DateTimeFormat { get; set; } = string.Empty;
    public string LatLngFormat { get; set; } = string.Empty;
    public ApplicationUser? User { get; set; }

    public int Zoom { get; set; } = 16;
    public double Lat { get; set; } = 53.88;
    public double Lng { get; set; } = 27.51;

    public int ShowNodesFromZoom { get; set; } = 16;
    public int SourceMapId { get; set; } = 1;

    public bool SwitchOffSuspicionSignalling { get; set; } = false;
    public bool SwitchOffRtuStatusEventsSignalling { get; set; } = false;

}

File: WebFt30/src/app/core/models/user-settings.ts
import { AppTheme } from '../store/settings/settings.state';

export class UserSettings {
  language!: string;
  theme!: AppTheme;
  dateTimeFormat!: string;
  latLngFormat!: string;
  zoom!: number;
  lat!: number;
  lng!: number;
  showNodesFromZoom!: number;
  sourceMapId!: number;
  switchOffSuspicionSignalling!: boolean;
  switchOffRtuStatusEventsSignalling!: boolean;
}
